ase/src/main/java/com/ase/ase/rest/response/TableDataPerDate.java

```diff
@@ -0,0 +1,39 @@
+package com.ase.ase.rest.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class TableDataPerDate {
```

Comment:
Rename this class. Call it what it is. It does not include table data but the hospitality situation.

Reply:
It does not only include hospital situation, also active cases and deaths. So maybe we can call it generalSituation or Summary or something like that